### ISSUE TITLE
Problem with image colors.

### ISSUE THREAD

**Issue by Steph55** [helix-toolkit/helix-toolkit#1883]
I'm trying to load different images with one of the examples in the toolkit folder, named ImageViewDemo. I notice that these images always appear darker than they should. I tried changing the light and other parameters but to no avail. I was wondering if some gamma adjustment is missing somewhere?

 I would need to use 3D images but with original colours, is this possible with the Helix Toolkit?

 By the way, apart from this, I find this toolkit really amazing, thanks a lot!

**Comment by holance** [comment]
Change the material to `DiffuseMaterial` and set following parameters:
```
            var white = new DiffuseMaterial()
            {
                DiffuseColor = Color.White,
                DiffuseMap = new MemoryStream(img.ToByteArray()),
                EnableUnLit = true
            };

            this.PlaneMaterial = white;
```

**Comment by Steph55** [comment]
Super! Thank you very much! I have one more question on this topic: is it
possible to keep the image transparency, like with png images?

**Comment by holance** [comment]
You can try with PNG, it should keep transparent.

**Comment by Steph55** [comment]
I just made other tests. I added a few objects and changed the light
direction, and the transparent pixels of the png images are still always
dark black...? There is no way I can see what is behind the image in those
transparent regions...

**Comment by holance** [comment]
In the example, it converts png to bitmap and set bitmap data into the texture. Bitmap does not support transparency. You need to use the actual png file path to create texture instead of the bitmap data.
```
            var white = new DiffuseMaterial()
            {
                DiffuseColor = Color.White,
                DiffuseMap = TextureModel.Create(imagepath),
                EnableUnLit = true
            };
```

**Comment by Steph55** [comment]
Thank you very much Lunci! However, with this change the transparent pixels
don't appear black anymore but they appear white and are still not
transparent. I tried to figure out where this white color is coming from
but I can't. I stepped through the code and I saw that the Texture2D is
effectively created with pixel format B8G8R8A8_UNorm but the MipsLevel is
1. I don't know if this is related to the problem...
